### PR TITLE
Added: Support for API v1.6. Defaults to v1.5 for compatibility.

### PR DESCRIPTION
**Pull request by ryanchausse**
Added support for Autotask API v1.6. URL changes are reflected in additional constants and default to v1.5 for backwards compatibility. Probably needs another commit for documentation prior to a tagged release.

### CODE REVIEW

**Comment by ryanchausse** [comment]
User would need to change constant AUTOTASK_API_VERSION from '1.5' to '1.6' in order to use API v1.6. Otherwise, no additional usage changes are needed. Need to update documentation to include "integrationcode" param from previous merged PR. Eventually, we can default to v1.6 and indicate the API version via a connect param as suggested by @mtingers

**Comment by MattParr** [comment]
Hi Ryan,
That is okay.  I will accept if you update the documentation.  The reason I have not done this as I said I would is I discovered I no longer have access to an Autotask instance that I can test against.  Infact, I only have one instance left.

As you can test it and construct the documentation as such, please update the documentation in this pull request and then I will merge.

**Comment by ryanchausse** [comment]
Hmm, usage will require an overrride of the version constant. Makes sense to simply include the connect param as discussed in this PR. Updating.

**Comment by ryanchausse** [comment]
Hi Matt,

It looks like there has been a change to .travis.yml generated and committed after running travis_pypi_setup.py and after my initial PR. Can you take a look please?

Thanks

**Comment by ryanchausse** [comment]
I have tested this on my instance of Autotask. Please let me know if you'd like me to change anything, and certainly feel free to squash!

**Comment by ryanchausse** [comment]
Just squashed it myself

File: atws/connection.py [hunk 1]
```diff
@@ -74,10 +81,15 @@ def get_connection_url(**kwargs):
     except (KeyError):
         url = None
     if not url:
-        url = get_zone_wsdl(kwargs['username'])
+        apiversion = 1.5
```
Comment (hoerup) [inline atws/connection.py]: you get an stacktrace if apiversion isn't specified 
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/atws/connection.py", line 86, in get_connection_url
    apiversion = kwargs['apiversion']
KeyError: 'apiversion'
wouldn't it be better to do something like 
if apiversion in kwargs:
 apiversion = kwargs['apiversion']
else:
 apiversion = 1.5
 logging.warning('apiversion not specified, defaulting to 1.5')

**Review by hoerup — CHANGES_REQUESTED** [review]

File: atws/connection.py (hunk 1 above)
Comment (ryanchausse) [reply atws/connection.py]: Right you are.

**Comment by hoerup** [comment]
Seems fine now

**Comment by ryanchausse** [comment]
Rebased ahead of merge

**Pull request merged**